PROTOCOL.md

```diff
@@ -227,10 +232,11 @@ represented by a space), and the rest of the string being their username.
 > Finish logging in (or renaming) by sending: `/trn USERNAME,0,ASSERTION`
 > where `USERNAME` is your desired username and `ASSERTION` is `data.assertion`.
 
-`|updateuser|USERNAME|NAMED|AVATAR|SETTINGS`
+`|updateuser|USERNAME@STATUS|NAMED|AVATAR|SETTINGS`
```

Comment:
```suggestion
`|updateuser|RANK_AND_USERNAME@STATUS|NAMED|AVATAR|SETTINGS`
```

though `RANK_AND_USERNAME` is ugly as sin, so if you have a better idea please change it, my point is that we still need to document the change somehow

Reply:
`USER` is a better call than `RANK_AND_USERNAME`, good judgement.